--- a/src/tui/tui.go
+++ b/src/tui/tui.go
@@ -678,6 +678,21 @@
 			leftMid:     ' ',
 			rightMid:    ' '}
 	}
+	if unicode && shape == BorderLeft {
+		return BorderStyle{
+			shape:       shape,
+			top:         '─',
+			bottom:      '─',
+			left:        '▌',
+			right:       '│',
+			topLeft:     '╭',
+			topRight:    '╮',
+			bottomLeft:  '╰',
+			bottomRight: '╯',
+			leftMid:     '├',
+			rightMid:    '┤',
+		}
+	}
 	if !unicode {
 		return BorderStyle{
 			shape:       shape,
